lxd/instance.go

```diff
@@ -204,6 +204,12 @@ func instanceCreateAsCopy(s *state.State, opts instanceCreateAsCopyOpts, op *ope
 		}
 	}
 
+	// Attempt to acquire exclusive access (succeeds if instance is stopped).
```

Comment:
I think we should have a comment here explaining why it is needed. 
And what happens if the instance is still running.